README.md

```diff
@@ -0,0 +1,19 @@
+# Corecito
+
+## Setting up
+
+*python3
+*To run this project, you may need to install the following python modules:
+
+```
+pip install cryptocom.exchange
+pip install import yaml

```

Comment:
This one didn't work completely for my, only the "import" library was installed. I got: 
```
$ pip3 install import yaml
Collecting import
  Using cached import-0.0.1.tar.gz (1.0 kB)
ERROR: Could not find a version that satisfies the requirement yaml (from versions: none)
ERROR: No matching distribution found for yaml
```

Reply:
Oops, it was a copy/paste mistake. I have updated it already as it should be:

`pip3 install yaml`